### ISSUE TITLE
Add setup instructions

### ISSUE THREAD

**Issue by Brecert** [supertiger1234/Nertivia-Server#1]
Currently, running your own development server is difficult and it's not documented how.
Adding setup instructions (for the database, etc..) would be helpful.

**Comment by supertiger1234** [comment]
Will do 👌

**Comment by kwantumhab** [comment]
Is there a config.js sample somewhere? 

I would like to try the server for myself but i cannot start it becouse of the lack of the config.js...(it this intentional?)

**Comment by supertiger1234** [comment]
It's kinda hard to set up because, you need Redis, MongoDB, Recaptcha token, Google API Token. I might someday post a sample config file, but not yet

**Comment by kwantumhab** [comment]
Shame... it looks promising... I just looking for some rocket.chat alternatives but id wanted to try this for private usege(own server, not public) I also tried to backtrack the config. classes from files; but no luck...

**Comment by supertiger1234** [comment]
Sorry :/

**Comment by NullRien** [comment]
i agree with the others, it needs a example-config.js and an example-fb-fcm.json with some instructions

**Comment by supertiger1234** [comment]
You can use these docker images made by cyber

Client: https://hub.docker.com/r/nertivia/client
Server: https://hub.docker.com/r/nertivia/server

**Comment by RyukoHQ** [comment]
docker images seems like deleted :/

**Comment by SupertigerDev** [comment]
Unfortunately, seems like the creator of this image has deleted all the work :/ I have no experience in docker. Sorry!
